zproject/backends.py

```diff
@@ -1303,11 +1303,23 @@ def dict_representation(cls, realm: Optional[Realm] = None) -> List[ExternalAuth
         ]
 
 
-def redirect_deactivated_user_to_login() -> HttpResponseRedirect:
+def redirect_to_signup(realm: Realm) -> HttpResponseRedirect:
+    signup_url = reverse("register")
+    redirect_url = realm.uri + signup_url
```

Comment:
Would we prefer to use `urljoin`s for such operations even on urls fully under our control?